packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -2604,9 +2596,9 @@ void AlgMJ<mj_scalar_t, mj_lno_t, mj_gno_t, mj_part_t, mj_node_t>::
   this->new_coordinate_permutations = Kokkos::View<mj_lno_t*, device_t>(
     Kokkos::ViewAllocateWithoutInitializing("num_local_coords"),
     this->num_local_coords);
-  // TODO empty is ok for NULL replacement?
+
```

Comment:
I'm not quite sure what the question means, but note that `view.data()` need not necessarily return `nullptr` if the view's dimensions are all zero.

Reply:
This was an old comment to myself during the refactor from raw ptrs to views - I should have deleted it sometime ago. I think I have ironed out all those issues and should be referencing just size in the logic. Thanks.